docs/en/observability/ci-cd-observability.asciidoc

```diff
@@ -1,7 +1,11 @@
 [[ci-cd-observability]]
-= CI/CD observability
+= CI/CD monitoring
```

Comment:
Should we include "Continuous Integration and Continuous Delivery (CI/CD)" in the title? It's a very common term so we probably don't need to, but I'm not sure where we draw the line on defining acronyms on the first mention. We _ spell out "Amazon Web Services (AWS)" in the title of that page. 🤷

Reply:
I think it wouldn't hurt and might help with SEO